lg_common/scripts/adhoc_browser.py

```diff
@@ -67,8 +67,8 @@ def main():
         adhocbrowser_viewport_publisher,
         viewport_name)
 
-    rospy.Subscriber('/director/scene', GenericMessage, adhocbrowser_director_bridge.translate_director)
-    rospy.Subscriber('/director/ready', Ready, adhocbrowser_pool.unhide_browsers)
+    rospy.Subscriber('director/scene', GenericMessage, adhocbrowser_director_bridge.translate_director)
+    rospy.Subscriber('director/ready', Ready, adhocbrowser_pool.unhide_browsers)
```

Comment:
does it make sense to use relative for /director/scene when that's sort of a global topic?